frontend-react/src/components/Admin/AdminFormEdit.tsx

```diff
@@ -92,8 +92,9 @@ export const TextAreaComponent = (params: {
                             // checkJson made sure the following JSON.parse won't throw.
                             params.savefunc(JSON.parse(text));
                         } else {
-                            showError(
+                            showToast(
                                 `JSON data generated an error "${errorMsg}"`,
```

Comment:
Should we also display the `text` here? would give more insight to the error possibly.

Reply:
The content of any current toast emissions should not have been touched. Any changes wanted on those should be done in another ticket.